externs/browser/html5.js

```diff
@@ -4166,29 +4166,17 @@ Navigator.prototype.plugins;
 Navigator.prototype.javaEnabled = function() {};
 
 /**
- * @type {Object}
+ * @type {!StorageQuota}
+ * @see https://developer.chrome.com/apps/offline_storage
  */
 Navigator.prototype.webkitPersistentStorage;
-/**
- * @param {number} newQuotaInBytes
- * @param {!Function=} opt_quotaCallback
- * @param {!Function=} opt_errorCallback
- *
- * @return {undefined}
- * @see https://developer.chrome.com/apps/offline_storage#method_overview
- */
-Navigator.prototype.webkitPersistentStorage.requestQuota = function(
-    newQuotaInBytes, opt_quotaCallback, opt_errorCallback) {};
 
 /**
- * @param {!Function=} opt_successCallback
- * @param {!Function=} opt_errorCallback
- *
- * @return {undefined}
- * @see https://developer.chrome.com/apps/offline_storage#method_overview
+ * @type {!StorageQuota}
+ * @see https://developer.chrome.com/apps/offline_storage
  */
-Navigator.prototype.webkitPersistentStorage.queryUsageAndQuota = function(
```

Comment:
No, `queryUsageAndQuota` is a member of `StorageQuota` defined in [fileapi.js](https://github.com/google/closure-compiler/blob/master/externs/browser/fileapi.js#L999)